### ISSUE TITLE
How to get DPM and SDP results for the training set?

### ISSUE THREAD

**Issue by YamacTan** [ifzhang/ByteTrack#275]
Hi, 

I just want to ask that what is the method for obtaining DPM and SDP results for the MOT17 Training set. It is necessary for uploading results to the MOTChallenge server, but the track.py script only works on the FRCNN detector when I use the train set. I didn't see any args or default code related to this. 
Any suggestions?

**Edit: I think the situation is probably due to COCO API. But still no solution found.**

**Comment by hijabfatima5** [comment]
Hi. can you please tell how did you train the model for MOT17 dataset?

**Comment by YamacTan** [comment]
So far I have not tried to perform the training process. @hijabfatima5

**Comment by Jacky-op-sm** [comment]
1. Only FRCNN in the track result:
in line 38 of convert_mot17_to_coco.py:
`if 'mot' in DATA_PATH and (split != 'test' and not ('FRCNN' in seq)):
       continue`

only the FRCNN is transformed into coco format, thus the track result on MOT17 training dataset only includes FRCNN.

2. To get tracking result on MOT17 test set
use whatever .txt file but name them with DPM and SDP on required files, since the MOT17 server only feedbacks the test dataset.
